Packs/PAN-OS/ReleaseNotes/1_17_6.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Updated the *pan-os-create-rule* and *panorama-create-rule* commands to support a comma-separated list of values for their **source_user** argument.
```

Comment:
```suggestion
- Updated the ***pan-os-create-rule*** command to support a comma-separated list of values for the *source_user* argument.
```

***panorama-create-rule*** is deprecated, no need to mention it in the RN.

Reply:
Oh that makes sense now, thats why only _**pan-os-create-rule**_ is in the README.md